bootstrapper/forms/formsNg2Bootstrapper.ts

```diff
@@ -42,6 +42,7 @@ export class FormsBootstrapper {
 	options: ITestItem[];
 	optionsAsync: Observable<ITestItem[]>;
```

Comment:
There is a naming convention I've seen others using and started using my self where observables are suffixed with `$` so in this case it would be `option$` I like this since it makes spotting observables very easy.